frontend/src/components/LoginForm/LoginForm.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { 
+  Button,
+  CardContent,
+  Checkbox,
+  IconButton,
+  InputAdornment,
+  Grid,
+  FormControlLabel,
+  Link,
+  Paper,
+  TextField,
+  Typography
+} from '@material-ui/core';
+import visibilityIcon from '../../assets/generic/visibility.svg';
+import visibilityOffIcon from '../../assets/generic/visibility-off.svg';
+import productIcon from "../../assets/generic/product-icon.svg"
+import { emailRegex, passwordRegex } from "../../constants/regex"
 
-const LoginForm = () => <h2>I&apos;m a login form, look at me</h2>;
+export default function LoginForm(props) {
```

Comment:
Wouldn't it be clearer if you used object destructuring here?

Reply:
@kibartas Will do that, chief

Reply:
@kibartas Changed it in another branch. Will create PR soon